cmd/skipper/main.go

```diff
@@ -29,8 +30,8 @@ var (
 )
 
 func main() {
-	cfg := config.NewConfig()
-	if err := cfg.Parse(); err != nil {
+	cfg := config.NewConfig("skipper")
```

Comment:
Why not `os.Args[0]`?

Reply:
I thought maybe it would be better to explicitly tell that that's the "skipper" or "routesrv" command regardless of how user names binary on his/her machine. But okay, I think it's more convenient to use `os.Args[0]`.